sklearn/feature_extraction/text.py

```diff
@@ -643,6 +643,18 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
 
         This is only available if no vocabulary was given.
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> #create document
+    >>> doc_test = ['Every summer our family gathers on the beach.',
+    ...             'Every winter our family gathers on the beach.',
```

Comment:
It would be good to have some word counted more than once, e.g. "We gather beach chairs on the beach."